CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 3.10.0-beta.1
+
+### Features
+
+- Introduced the ability to change the [capture mode](https://api-references.dolby.io/comms-sdk-flutter/dolbyio_comms_sdk_flutter/LocalAudio/setCaptureMode.html) in non-Dolby Voice conferences and use either the [Standard](https://api-references.dolby.io/comms-sdk-flutter/dolbyio_comms_sdk_flutter/AudioCaptureMode.html#standard) or [Unprocessed](https://api-references.dolby.io/comms-sdk-flutter/dolbyio_comms_sdk_flutter/AudioCaptureMode.html#unprocessed) mode.
+
+- Introduced a new [voiceFont](https://api-references.dolby.io/comms-sdk-flutter/dolbyio_comms_sdk_flutter/AudioCaptureOptions/voicefont.html) property to [AudioCaptureOptions](https://api-references.dolby.io/comms-sdk-flutter/dolbyio_comms_sdk_flutter/AudioCaptureOptions-class.html) that is supported in the [Standard](https://api-references.dolby.io/comms-sdk-flutter/dolbyio_comms_sdk_flutter/AudioCaptureMode.html#standard) audio capture mode. [Voice fonts](https://api-references.dolby.io/comms-sdk-flutter/dolbyio_comms_sdk_flutter/AudioCaptureOptions/voicefont.html) allow participants to modify their voices in real time to improve social interaction in entertainment scenarios. For more information about this feature, see the [Using Voice Fonts](https://docs.dolby.io/interactivity/docs/guides-using-voice-fonts) guide.
+
+- Introduced a new [preview](https://api-references.dolby.io/comms-sdk-flutter/dolbyio_comms_sdk_flutter/LocalAudio/preview.html) service that allows the local participant to test different capture modes and voice fonts before a conference. The service sets a preview recorder that records the participant's audio and plays it back. Before playing the recorded audio, set the [captureMode](https://api-references.dolby.io/comms-sdk-flutter/dolbyio_comms_sdk_flutter/LocalAudio/setCaptureMode.html) to a preferred setting that you wish to try.
```

Comment:
Perhaps 

`The service records the local participant's audio and plays it back thereafter.`

it's a bit more accurate then

`The service sets a preview recorder that records the participant's audio and plays it back.`

Also perhaps

`Before recording the audio for previewing, set the ...`

in stead of:

`Before playing the recorded audio, set the ...`

Reply:
Are you sure? I think that Yuriy explained to me some time ago that you record your natural voice and then, before playing it, you can change it using setCaptureMode.

Reply:
The flow we use for testing is:
1. select voice font
2. start recording
3. (optional) cancel 
4. play recording
5. release recording

Also to change a bit the test I wrote:

`The service records the local participant's audio and allows the recording to be played back thereafter.`

This in stead of:

`The service records the local participant's audio and plays it back thereafter.`

Reply:
Ok. Just tested agin. It works as you explained as well.

Reply:
Since is works both ways we can leave it as is.